app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -20,6 +20,8 @@ def ems_inv_to_hashes
     get_vdcs
     get_vapps
     get_vms
+    get_private_images if @options.get_private_images
```

Comment:
Is it possible to link a vm to its image?  Similar to https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/google/cloud_manager/refresh_parser.rb#L227
If so we'll want to get images before vms.